MTLManagedObjectAdapter.xcodeproj/project.pbxproj

```diff
@@ -569,40 +592,44 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 5470B58C198514F700E4D941 /* Release.xcconfig */;
 			buildSettings = {
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = NO;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
This looks like it already drops support for 10 and 11. Is https://ovuline.atlassian.net/browse/POD-8610 in the queue?

Reply:
Looks like it's still in to be groomed. POD-8612 has also been moved to next sprint due to other priorities needing to be handled in the current sprint. It shouldn't be an issue to handle POD-8610 before it if we have to.